src/sections/Navigation/utility/data.js

```diff
@@ -39,19 +39,19 @@ const Data = {
             subItems: [
                 {
                     name: "Meshery",
-                    path: "#"
+                    path: "/projects/meshery"
```

Comment:
How do we ensure that the `/meshery` path works as well?

Reply:
We can have redirect paths in the mdx file for meshery and others, similar to the one we do in jekyll.

Reply:
Sounds real good